username_api.py

```diff
@@ -12,11 +14,56 @@
 
 patterns = yaml.load(open('websites.yml'))
 
-def check_username(website, username):
-	url = patterns['urls'].get(website, 'https://{w}.com/{u}').format(
+def get_profile_url(website, username):
+	return patterns['urls'].get(website, 'https://{w}.com/{u}').format(
 		w=website,
-		u=username
-	)
+		u=username)
```

Comment:
unnecessary change here

Reply:
This is actually not a very good diff comparison, I have a fresh function in those lines, for some reason because of the repetition from the old one, git decides it's a modification of that. I'll change it nonetheless.

Reply:
it is a very good comparison. :P
Splitting a function like this is very common.
not changing the lines in the middle helps code review and understanding what is changing and what isnt.